src/BookNavigator/book-navigator.js

```diff
@@ -71,7 +71,10 @@ export class BookNavigator extends LitElement {
       'volumes',
       'chapters',
       'search',
-      'bookmarks'
+      'downloads',
+      'bookmarks',
+      'visualAdjustments',
+      'share'
```

Comment:
potential blocker - bc we have these newly listed items here - do they show up as shortcuts?

Reply:
we might not need downloads, visualadjustments, share here bc we don't surface them in shortcuts view - let's just remove them.